.github/workflows/mergify-ready.yml

```diff
@@ -26,3 +28,52 @@ jobs:
             needs ${{ toJSON(needs.pre_check.outputs) }}
           EOF
           sleep 5
+
+  merge-strategy:
+    runs-on: ubuntu-latest
+    if: >-
+      github.event_name != 'pull_request' ||
+      github.event.pull_request.draft == true ||
+      github.event.pull_request.base.ref != 'master' || (
+        contains(github.event.pull_request.labels.*.name, 'automerge:squash') || 
+        contains(github.event.pull_request.labels.*.name, 'automerge:merge') ||
```

Comment:
FWIW, I'd like to rename `automerge:merge` to `automerge:expert`, in accordance with earlier discussions how it might otherwise be overused by people who may not know its nuances.

Reply:
Or `automerge:no-update`.

Reply:
Or `automerge:manual-updates`.

Reply:
I already updated the label description to indicate it's an expert option, and I'm hoping the new no-merge-commits check will help reduce it's usage (it should bump it off the queue if a merge commit is added to make the PR up to date)

Reply:
Out of these I prefer the `automerge:no-updates`